requirements-dev.txt

```diff
@@ -16,7 +16,7 @@ matplotlib
 nbsphinx
 pydocstyle
 pytest >=3.2
-sphinx >=3.4
+sphinx ==4.5
```

Comment:
just curious, why 4.5?

Reply:
i copied the pin from `compas`, but that makes little sense, I updated to match `compas_timber`'s pin